src/server/ua_services_attribute.c

```diff
@@ -411,9 +411,7 @@ void Service_Read(UA_Server *server, UA_Session *session, const UA_ReadRequest *
 		size_t offset = 0;
         UA_Connection *c = UA_NULL;
         UA_SecureChannel *sc = session->channel;
-        if(sc)
```

Comment:
Reverted, FIXME added and issue https://github.com/acplt/open62541/issues/354 opened.